### ISSUE TITLE
Source request (EN): Mangakakalot.to

### ISSUE THREAD

**Issue by Zentberk919** [keiyoushi/tachiyomi-extensions-source#5361]
### Source name

Mangakakalot.to

### Source link

https://mangakakalot.to/home

### Source language

English

### Other details

this site is different from mangakakalot.com/chapmanganato.to
so i request new source

and i have been forwarded to post a source request
https://discord.com/channels/1193460528052453448/1193504423964586025/1291758826718232710

### Acknowledgements

- [X] I have checked that the extension does not already exist by searching the [extension listing](https://keiyoushi.github.io/extensions/) and verified it does not appear there.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [x] I will fill out all of the requested information in this form.

**Comment by MikeZeDev** [comment]
Its just hot-linking picture from Mangareader.to cdn. Its useless to add this website i think.

![Image](https://github.com/user-attachments/assets/2fe53ae4-d5d6-43c7-bca9-b12535c6d5e7)

**Comment by Norsze** [comment]
Yeah this site currently has no content, while MangaReader source server is down. There is no need for this extension. Use the MangaReader extension, same content, same servers.
